jest.config.js

```diff
@@ -26,7 +26,7 @@ module.exports = {
 
   collectCoverage: true,
   coverageDirectory: `./test/unit/coverage`,
-  coverageReporters: [`lcov`],
+  coverageReporters: [`lcov`, `text-summary`], //change 'text' to 'text-summary' for summarized output
```

Comment:
let's remove the misguiding comment I propose.

Plus the changelog is missing.